docs/userguide/dev/ci.md

```diff
@@ -0,0 +1,170 @@
+3. "Branch protection rules"
+4. Choose one of the rule sets and click "Edit".  ***Note: all rule sets should be changed in the same way***
+5. Ensure that "Require status checks to pass before merging" is enabled
+6. Underneath that enabled checkbox, the table "Status checks that are required." lists the currently enabled tests ("checks") that must pass before a Pull Request can be merged to the branch
+7. If a new test (check) needs to be added, use the search box above the table and type the display name of the check, then click on it in the drop down list to add it to the table of required checks.
+
+
+## Github Actions Configuration
+
+ICU uses [Github Actions](https://github.com/features/actions) to run unit tests, environment tests, deployments of the User Guide, etc.
+
+### Workflow files
+
+Workflow files are in YAML form stored at `.github/workflows/`.
+
+### UI Dashboard
+
+All workflow instances/runs show up in the ["Actions" tab](https://github.com/unicode-org/icu/actions) of the repository,
+using the display name that was set for the workflow.
+All of the runs can be filtered by the Workflow name and/or by the Pull Request (via the Branch filter).
+
+Each PR will show the status of the current/latest run of each job in the "Checks" section of the Pull Request's main page (Conversation tab), 
+with special details and logs being navigable in the Checks tab.
+
+### Configuration
+
+#### Jobs
+
+Jobs can be added, removed, and edited merely by editing the YAML workflow files.
+
+#### Conditional Triggers
+
+Workflow files can be conditionally triggered based on many factors.
+A few of the useful factors:
+
+* whether the trigger event is a push to a pull request, or the merge of a pull request, or a manual request
+* the name of the originating branch for the trigger event
+* whether the files modified in a pull request belong to certain file paths
+
+ICU uses the file path matching condition to run ICU4J or ICU4C tests respectively only when their corresponding directories `icu4j` and `icu4c` contain file changes.
+Thus, there is a separate workflow for ICU4J and ICU4C,
+each with its own separate set of trigger conditions based on their corresponding file paths.
+User Guide deployments are similarly run conditionally only when the `docs` directory has changed.
+Some tests that are commonly needed or cross boundaries of concern are always run.
+
+Some long-running tests are run once a week using a cronjob-like schedule trigger.
+
+Some workflows exist to interact with the Github Actions platform,
+such as keeping the cache alive for Maven artifacts due to the [noticeable flakiness that PRs face](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows) without the cache. Because a [cache eviction policy](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy), which affects any subsequent PRs until they are merged,
```

Comment:
```suggestion
such as keeping the cache alive for Maven artifacts due to the [noticeable flakiness that PRs face](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows) without the cache.
Because of a [cache eviction policy](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy), which affects any subsequent PRs until they are merged,
```